input/pagecontent/4-2-syncconsiderations.md

```diff
@@ -81,6 +86,7 @@ There are in some cases good reasons for a client not to follow the subscribed c
 
 Many applications go into edit mode or start a modal dialog that locks the system from changing context without user intervention. Examples can be when modifying texts, reports, annotating images or performing administrative tasks. The clients may then decline to follow the subscribed context to prevent loss of end user data.
 
+{:.grid}
 |System|Failure mode|Possible actions|
 |--|--|--|
 | Subscribing Client | Modal dialog open in UI, unable to change case without losing end user data | Present end user with clear indication that contextual synchronization is lost. Respond with a http status code of 409 conflict. |

```

Comment:
In a below Subscribing Client row, "Text change is refused, stating the source and reason for change" should be "Context change is refused, stating the source and reason for change"

Reply:
Fixed.